simp_config_answers.yaml

```diff
@@ -66,47 +54,47 @@ gateway: "10.0.2.2"
 # convert your system to a caching DNS server. You'll know
 # that you have this situation if you end up with a duplicate
 # definition for File['/etc/named.conf'].
-"dns::servers":
-  - "192.168.33.10"
-  - "10.0.2.3"
-  - "8.8.8.8"
+dns::servers:
+  - '192.168.33.10'
```

Comment:
It is a recursive; it should probably be a forwarding server instead.  Either way, we want all clients performing all DNS through the server so they can consistently resolve names in the `simp.test` domain.

Making the server a forwarder down the line will be a requirement, I think, because a lot of networks restrict DNS traffic.  It will need to accept a server or a set of servers as arguments or otherwise be configurable from the Vagrantfile repo.  For now, though, I'm mostly focused on getting this working in general, with an occasional low-effort side project.